### DISCUSSION TITLE
session_set_cookie_params?

### DISCUSSION THREAD

**Discussion by skweeker** [magicbug/Cloudlog#1428] [Q&A] [answered]
Hey all! Getting back into things here... I've got a container based off of Alpine linux. The install loads fine, but then after I run the install, I am getting an error when I hit the root level index.php
```
An uncaught Exception was encountered

Type: Error

Message: Call to undefined function session_set_cookie_params()

Filename: /var/www/localhost/htdocs/system/libraries/Session/Session.php

Line Number: 291
```

I did check Session.php, and indeed the function is not defined in there. I poked around a little bit, and didn't find it defined anywhere else, either. I'm positive something is goobered in my environment, but I'm not sure where to start looking. Might someone have some thoughts on this? I just cloned the repo tonight.

73!

**Comment by AndreasK79** [comment, marked as answer]
@skweeker this is a PHP function. 

Some help might be here: https://stackoverflow.com/questions/17500019/call-to-undefined-session-set-cookie-params

**Reply by skweeker** [reply]
Okay! So, I ended up having to install the module for this, which for me was 'php7-session' and I also had to install 'php7-ctype' for a subsequent error, but now I'm at the Cloudlog UI. Yay! Now to figure out why it's showing no contacts on an existing database... but that's a separate issue! Thanks!

**Reply by AndreasK79** [reply]
Check the table to see if the station_id is set.
